newIDE/app/src/MainFrame/Toolbar/PreviewAndPublishButtons.js

```diff
@@ -0,0 +1,192 @@
+                  )
+                }
+                exceptionalTooltipForToolbar={<Trans>{previewTooltip}</Trans>}
+              />
+            }
+            openMenuWithSecondaryClick
+            buildMenuTemplate={(i18n: I18nType) => [
+              {
+                label: i18n._(t`Launch another preview in a new window`),
+                click: onPreviewWithoutHotReload,
+                enabled: isPreviewEnabled && hasPreviewsRunning,
+              },
+              { type: 'separator' },
+              ...(previewState.overridenPreviewLayoutName
+                ? [
+                    {
+                      type: 'checkbox',
+                      label: previewState.overridenPreviewExternalLayoutName
+                        ? i18n._(
+                            t`Start all previews from external layout ${
+                              previewState.overridenPreviewExternalLayoutName
+                            }`
+                          )
+                        : i18n._(
+                            t`Start all previews from scene ${
+                              previewState.overridenPreviewLayoutName
+                            }`
+                          ),
+                      checked: previewState.isPreviewOverriden,
+                      click: () =>
+                        setPreviewOverride({
+                          isPreviewOverriden: !previewState.isPreviewOverriden,
+                          overridenPreviewLayoutName:
+                            previewState.overridenPreviewLayoutName,
+                          overridenPreviewExternalLayoutName:
+                            previewState.overridenPreviewExternalLayoutName,
+                        }),
+                    },
+                    { type: 'separator' },
+                  ]
+                : []),
+              {
+                label: previewState.previewExternalLayoutName
+                  ? i18n._(
+                      t`Use this external layout inside this scene to start all previews`
+                    )
+                  : i18n._(t`Use this scene to start all previews`),
+                click: () =>
+                  setPreviewOverride({
+                    isPreviewOverriden: true,
+                    overridenPreviewLayoutName: previewState.previewLayoutName,
+                    overridenPreviewExternalLayoutName:
+                      previewState.previewExternalLayoutName,
+                  }),
+                enabled:
+                  previewState.previewLayoutName !==
+                    previewState.overridenPreviewLayoutName ||
+                  previewState.previewExternalLayoutName !==
+                    previewState.overridenPreviewExternalLayoutName,
+              },
+            ]}
+          />
+          <FlatButton
+            onClick={exportProject}
+            disabled={!hasProject}
+            icon={
+              <img
+                alt={t`Publish`}
```

Comment:
This won't work, you must use i18n._ to transform `t` (which gives an object) to a string